translations/pl.ts

```diff
@@ -1034,11 +1034,11 @@
     </message>
     <message>
         <source>feet</source>
-        <translation type="unfinished"></translation>
+        <translation>stopa</translation>
```

Comment:
Here should be plural. It is used in various contexts, it is part of the utility method. For example during navigation: "in 300 feet, turn right"... I understand polish just a little, but it sounds me as a singular :-)

Reply:
Because it's singular :) I will fix it.
It's always good to know the context because in Polish plural form of feet is "stopy" but "in 300 feet, turn right" will be " za 300 stóp, skręć w prawo"